flocker_bb/ec2.py

```diff
@@ -82,17 +90,10 @@ def symbol():
 StopFailed = trivialInput(Input.STOP_FAILED)
 
 
-@attributes([
-    'name',
-    'image_name',
-    'size',
-    'keyname',
-    'security_groups',
-    'userdata',
-    'instance_tags',
-    'image_tags',
-], apply_immutable=True)
-class EC2(object):
+@attributes(['driver'], apply_immutable=True)
+class InstanceBooter(object):
```

Comment:
Because of this renaming and the generalization to support more than EC2, we want to rename this module too.  We didn't want to do it before the PR was reviewed in case it made the diff too hard to read.